### ISSUE TITLE
generalize 'ampcalsrc' to 'ampcalsrcs' in EXP.yaml

### ISSUE THREAD

**Issue by dingswin** [dingswin/psrvlbireduce#6]
Sometimes in one observation there are more than one fringe finder, to tackle that we need to generalize ampcalsrc setup to a dictionary called ampcalsrcs, which specifies the antennas each fringe finder is responsible for.

**Comment by adamdeller** [comment]
Concur, although that will require some quite a bit of extra configuration and implementation.  Currently, the pipeline tries to use just one scan of 1 source for the time-invariant delay calibration and bandpass calibration.  Being able to 1 scan for a subset of antennas, and then another scan (potentially on another source) for other antennas, would be great - but you'll need to implement this in both the amplitude calibrate FRING step, and the BPASS step.

**Comment by dingswin** [comment]
Then we might need to modulize amplitude calibration and BPASS first before implementation of this new feature. That would make the implementation much tidier.

**Comment by adamdeller** [comment]
Yes - the important thing will be to make sure that antennas not designated to be corrected in a given scan are passed through uncorrected, rather than dropped, and that all antennas are corrected in at least one scan.
